### DISCUSSION TITLE
Resuming previewer in the exact state as last left it

### DISCUSSION THREAD

**Discussion by abkrm** [ibhagwan/fzf-lua#1240] [Q&A] [answered]
Hello! :)
I couldn't find any info on it other than what the wiki has to say about "fzf-lua".resume(), but that didn't lead me to a solution. Basically I would like the resume provider to resume at the exact state as I last left it (order of results, the line my cursor was in the previewer etc.,). Is this currently possible with fzf-lua? If yes, how can I achieve this?
(I am trying to replicate here what Telescope's resume feature does basically)

**Comment by ibhagwan** [comment]
Unfortunately due to the limitations of using an external process it is not possible ATM.

I have plans to explore hiding the windows instead of closing thus being able to return to the exact position, you can follow #1229 to get notified on progress on that front.

**Comment by ibhagwan** [comment, marked as answer]
This is now possible with  the addition of the `hide` profile, to test, run `:FzfLua profiles load=hide`, see the README resume section to see how to setup permanently.

**Reply by nadiamoe** [reply]
This is awesome, thank you!
